rskj-core/src/main/resources/reference.conf

```diff
@@ -334,7 +334,7 @@ sync {
 
     snapshot = {
         # Client / snapshot sync enabled (Server is always enabled)
-        enabled = false
+        enabled = true
```

Comment:
Are we going to leave this ON by default?

Reply:
changed it to false